README.md

```diff
@@ -4,17 +4,14 @@ This project demonstrates how [TypeORM](https://github.com/typeorm/typeorm) can
 ### How to run this example
 1. Install the cordova cli: `npm install -g cordova`
 2. Install all dependencies: `npm install`
-3. Copy `Reflect.js` (`node_modules/reflect-metadata`) and `system.src.js` (`node_modules/systemjs/dist`) into the `www/lib` folder
-4. Copy `typeorm-browser.js` and `typeorm-browser.js.map` (`node_modules/typeorm`) into `www/lib/typeorm`
-5. Compile: `tsc`
-6. Add your desired platform: `cordova platform add <ios | android>`
-7. Run the app: `cordova run <ios | android>`
+3. Compile: `tsc`
+4. Pack js files `webpack`
```

Comment:
Please add a colon before `webpack`.

Reply:
Also, this would require that you have webpack installed globally, right?

Reply:
Changed to use npm script and local webpack.